source/blog/2016-11-02-ovirt-provider-ovn.md

```diff
@@ -51,6 +51,7 @@ After installing the driver and OVN, the OVN-controller must be configured. This
     vdsm-tool ovn-config <OVN central server IP> <local IP used for OVN tunneling>
 
 The second parameter (local IP used for OVN tunneling) can be the IP address of the ovirtmgmt interface on the host, and should be reachable by OVN hosts and central server.
+vdsm-tool will also configure ovn-controller with the PKI required to safely connect to the ovn central server.
```

Comment:
Maybe in a new paragraph:
"To add support for Transport Layer Security (TLS) connectivity with the OVN-controller, by default, vdsm-tool creates private and public keys, and configures OVN to enable TLS encryption between the host and the central OVN-controller."

Marcin, do you have a patch which includes the changes in the script?